spateo/tools/cluster_degs.py

```diff
@@ -11,13 +11,11 @@
 from scipy.stats import mannwhitneyu
 from sklearn.neighbors import NearestNeighbors
 from statsmodels.sandbox.stats.multicomp import multipletests
+from sympy import N
 from tqdm import tqdm
 from typing_extensions import Literal
 
-from ..configuration import SKM
 
-
-@SKM.check_adata_is_type(SKM.ADATA_UMI_TYPE, optional=True)
```

Comment:
Joseph suggested to add this adata validator.  See this merged pull request: https://github.com/aristoteleo/spateo-release/pull/102. 

Please add this back and use it in future functions